cypress/e2e/exampleTest.cy.js

```diff
@@ -1,8 +1,48 @@
-describe ('sample test', () => {
+import { aliasQuery, aliasMutation, hasOperationName } from '../utils/graphql-test-utils'
 
-  it('Should be able to pass a test', () => {
+describe ('Dashboard', () => {
+
+  it('Should display homepage information', () => {
+    cy.intercept('POST', 'https://immense-hollows-78338.herokuapp.com/graphql', (req) => {
```

Comment:
I'm so glad that Dara helped us figure out the order of intercepting/visiting!

Reply:
me too!